lstchain/reco/dl1_to_dl2.py

```diff
@@ -523,18 +530,18 @@ def apply_models(dl1,
     Parameters
     ----------
     dl1: `pandas.DataFrame`
-    classifier: Random Forest Classifier
-        RF for Gamma/Hadron separation
-    reg_energy: Random Forest Regressor
-        RF for Energy reconstruction
-    reg_disp_vector: Random Forest Regressor
-        RF for disp vector reconstruction
-    reg_disp_norm: Random Forest Regressor
-        RF for disp norm reconstruction
-    cls_disp_sign: Random Forest Classifier
-        RF for disp sign reconstruction
+    classifier: string or Random Forest Classifier
+        Location of the, or pre-loaded, RF for Gamma/Hadron separation
```

Comment:
unclear docstring to me.
type should be string | Path | bytes | `sklearn.ensemble.RandomForestClassifier`
and path to the random forest filename or file or pre-loaded RandomForestClassifier object as a description.